ui/tests/integration/components/mount-backend-form-test.js

```diff
@@ -206,22 +206,21 @@ module('Integration | Component | mount backend form', function (hooks) {
         await render(
           hbs`<MountBackendForm @mountType="secret" @mountModel={{this.model}} @onMountSuccess={{this.onMountSuccess}} />`
         );
-        for (const engine of WIF_ENGINES) {
```

Comment:
I'd actually argue this is a good place for this iteration because this **is** a place where the `type` changes the form fields rendered